src/placement/jump_map.c

```diff
@@ -286,12 +287,14 @@ static int
 obj_remap_shards(struct pl_jump_map *jmap, uint32_t layout_ver, struct daos_obj_md *md,
 		 struct pl_obj_layout *layout, struct jm_obj_placement *jmop,
 		 d_list_t *remap_list, d_list_t *out_list, uint32_t allow_status,
-		 uint8_t *tgts_used, uint8_t *dom_used, uint8_t *dom_full,
-		 uint32_t failed_in_layout, bool *is_extending, uint32_t fdom_lvl)
+		 uint32_t allow_version, uint8_t *tgts_used, uint8_t *dom_used,
```

Comment:
The parameters in the comment block above this function seems to be out of date now.

Reply:
sure, will update.